<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="ThreeDWindow" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="BTN_set_origin">
          <property name="text">
           <string>Set Origin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_x_plus">
          <property name="text">
           <string>X+90</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_y_plus">
          <property name="text">
           <string>Y+90</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_z_plus">
          <property name="text">
           <string>Z+90</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="BTN_StartAllLpms">
        <property name="text">
         <string>Connect All</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="load_choose1">
          <property name="text">
           <string>&lt;2 kg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="load_choose2">
          <property name="text">
           <string>2~10 kg intermittent</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="load_choose3">
          <property name="text">
           <string>2~10 kg repeated</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="load_choose4">
          <property name="text">
           <string>&gt;10 kg</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="check_rula_1">
        <property name="text">
         <string>shoulder is raised</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_rula_2">
        <property name="text">
         <string>upper arm is abducted</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_rula_3">
        <property name="text">
         <string>arm is supported or person is leaning</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_rula_4">
        <property name="text">
         <string>hold &gt; 10 mins or 4 times/min</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Wrist &amp; Arm Score</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Label_WristArmScore">
          <property name="font">
           <font>
            <family>Adobe Devanagari</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="textbrowser"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1233</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuwindow">
    <property name="title">
     <string>window</string>
    </property>
    <addaction name="actionclose"/>
    <addaction name="separator"/>
    <addaction name="OpenHead"/>
    <addaction name="OpenRightUpperArm"/>
    <addaction name="OpenRightLowerArm"/>
    <addaction name="OpenRightWrist"/>
    <addaction name="OpenLeftUpperArm"/>
    <addaction name="OpenLeftLowerArm"/>
    <addaction name="OpenLeftWrist"/>
    <addaction name="OpenBody"/>
    <addaction name="actionLeft_Upper_Leg"/>
    <addaction name="actionLeft_Lower_Leg"/>
    <addaction name="actionRight_Upper_Leg"/>
    <addaction name="actionRight_Lower_Leg"/>
   </widget>
   <addaction name="menuwindow"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionclose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="OpenHead">
   <property name="text">
    <string>Head</string>
   </property>
  </action>
  <action name="OpenRightUpperArm">
   <property name="text">
    <string>Right Upper Arm</string>
   </property>
  </action>
  <action name="OpenRightLowerArm">
   <property name="text">
    <string>Right Lower Arm</string>
   </property>
  </action>
  <action name="OpenRightWrist">
   <property name="text">
    <string>Right Wrist</string>
   </property>
  </action>
  <action name="OpenLeftUpperArm">
   <property name="text">
    <string>Left Upper Arm</string>
   </property>
  </action>
  <action name="OpenLeftLowerArm">
   <property name="text">
    <string>Left Lower Arm</string>
   </property>
  </action>
  <action name="OpenLeftWrist">
   <property name="text">
    <string>Left Wrist</string>
   </property>
  </action>
  <action name="OpenBody">
   <property name="text">
    <string>Body</string>
   </property>
  </action>
  <action name="actionLeft_Upper_Leg">
   <property name="text">
    <string>Left Upper Leg</string>
   </property>
  </action>
  <action name="actionLeft_Lower_Leg">
   <property name="text">
    <string>Left Lower Leg</string>
   </property>
  </action>
  <action name="actionRight_Upper_Leg">
   <property name="text">
    <string>Right Upper Leg</string>
   </property>
  </action>
  <action name="actionRight_Lower_Leg">
   <property name="text">
    <string>Right Lower Leg</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ThreeDWindow</class>
   <extends>QWidget</extends>
   <header>threedwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
